service/src/main/resources/db/migration/V294__service_worker_message_account.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE message_account DROP COLUMN type CASCADE;
+ALTER TABLE message_account ADD COLUMN type message_account_type;
```

Comment:
Is dropping and re-adding actually needed? The column already exists and is the correct type.

Reply:
It's needed because it seems that `GENERATED ALWAYS` cannot be dropped.